README.md

```diff
@@ -125,6 +125,12 @@ crates through the gateway.
 
 ## Examples
 
+The following example is a template for bootstrapping a new bot using
+Twilight's HTTP and gateway clients with its in-memory cache. In order to
+run this, replace the contents of a new project's `main.rs` file with the
+following. Be sure to set the `DISCORD_TOKEN` environment variable to your
+bot's token.
+
```

Comment:
@7596ff outside of the rest of the changes that the other PR seems to cover, this little blurb would have helped me a bit to get started. But I'll leave it up to you.

Reply:
Okay, can you remove the conflicting parts of the file, and then we can merge it?